hack/e2e-cleanup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -xe
+
+# Remove SBR finalizers
+SBR=$(kubectl get sbrs)
+if [ $? -eq 0 ]; then
```

Comment:
no need to `if` check - invoking bash with `-xe` option will immediately exit the shell in case of non zero exit code on any subcommands

see https://github.com/redhat-developer/service-binding-operator/pull/639#discussion_r478371705

Reply:
I've missed that... Thx